### PR TITLE
Support status for 15 SP3 is updated (jsc#sle 17766)

### PR DESCRIPTION
**Pull request by tbazant**
### Are there any relevant issues/feature requests?

* jsc#SLE-17766

### To which product versions do the changes apply?

The first column is to be filled by the requester, the second column is to be filled by the doc team after the changes have been backported to the maintenance branches.

| Code 15     | Backport Done
| ----------  | ----------
| [x] 15 SP3  | *(no backport necessary)*
| [ ] 15 SP2  | [ ]
| [ ] 15 SP1  | [ ]
| [ ] 15 SP0  | [ ]

| Code 12     | Backport Done
| ----------  | ----------
| [ ] 12 SP5  | [ ]
| [ ] 12 SP4  | [ ]
| [ ] 12 SP3  | [ ]
| [ ] 12 SP2  | [ ]

### CODE REVIEW

File: xml/virt_support.xml [hunk 1]
```diff
@@ -100,105 +100,120 @@
...
+     <colspec colnum="1" colname="1"/>
+     <colspec colnum="2" colname="2"/>
+     <tbody>
+      <row>
+       <entry><para>Maximum virtual CPUs per VM</para></entry>
+       <entry><para>256</para></entry>
+      </row>
+      <row>
+       <entry><para>Maximum memory per VM</para></entry>
+       <entry><para>4&nbsp;TiB</para></entry>
```
Comment (aginies) [inline xml/virt_support.xml:145]: its 6Tib.
https://jira.suse.com/browse/SLE-17724

Comment (tbazant) [inline xml/virt_support.xml:145]:
```suggestion
       <entry><para>6&nbsp;TiB</para></entry>
```

**Review by tbazant — COMMENTED** [review]
fixed value as suggested

File: xml/virt_support.xml [hunk 2]
```diff
@@ -250,26 +265,24 @@
...
    <title>Since &sls; 12 SP3, all the following guest operating systems are fully supported (L3):</title>
    <listitem><para>&sls; 11 SP4</para></listitem>
    <listitem><para>&sls; 12 SP3, SP4, SP5</para></listitem>
-   <listitem><para>&sls; 15 SP1, SP2, SP3</para></listitem>
-   <listitem><para>&sled; 15 SP3</para></listitem>
+   <listitem><para>&sls; 15 GA, SP1, SP2, SP3</para></listitem>
    <listitem><para>Windows Server 2008 SP2+, 2008 R2 SP1+, 2012+, 2012 R2+, 2016, 2019</para></listitem>
+   <listitem>
+    <para>
+     Oracle Linux 6,7,8
```
Comment (alice-suse) [inline xml/virt_support.xml:278]: @tbazant According to https://jira.suse.com/browse/SLE-17766 description, oracle only plans to support 3rd party kvm vendor, which does not include xen. Right?

**Pull request merged**